core/src/main/java/de/danoeh/antennapod/core/syndication/namespace/NSMedia.java

```diff
@@ -51,25 +67,50 @@ public SyndElement handleElementStart(String localName, HandlerState state,
 				String durationStr = attributes.getValue(DURATION);
 				if (!TextUtils.isEmpty(durationStr)) {
 					try {
-                        long duration = Long.parseLong(durationStr);
+						long duration = Long.parseLong(durationStr);
```

Comment:
The file is tabbed anyway, all but this line. The only thing worse than tabs, is mixing.

Android Studio, Ctrl + Alt + L did convert this to spaces, but the result was every line changed and it's not my place to do that. I think you guys should do that against the whole project, the result being the project being all spaces and unified. But that's just my opinion.